docs/guide/use_cases/self_managed_lb/index.md

```diff
@@ -0,0 +1,109 @@
+- Port and Protocol of Target [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+
+## Configure Load Balancer
+
+**Create Load Balancer: (optional)**
+
+1. Use the [create\-load\-balancer](https://docs.aws.amazon.com/cli/latest/reference/elbv2/create-load-balancer.html) command to create an IPv4 load balancer, specifying a public subnet for each Availability Zone in which you launched instances. 
+
+    `kubernetes.io/cluster/your-cluster-name`
+
+    You can specify only one subnet per Availability Zone. 
+
+    ```
+    aws elbv2 create-load-balancer --name my-load-balancer --type network --subnets subnet-0e3f5cac72EXAMPLE
+    ```
+
+    **Important:** The output includes the ARN of the load balancer. This value is needed to configure the LBC. 
+
+    Example:
+
+    ```
+    arn:aws:elasticloadbalancing:us-east-2:123456789012:loadbalancer/net/my-load-balancer/1234567890123456
+    ```
+
+
+1. Use the [create\-target\-group](https://docs.aws.amazon.com/cli/latest/reference/elbv2/create-target-group.html) command to create an IPv4 target group, specifying the same VPC of your EKS cluster. 
+
+   ```
+   aws elbv2 create-target-group --name my-targets --protocol TCP --port 80 --vpc-id vpc-0598c7d356EXAMPLE
+   ```
+
+   The output includes the ARN of the target group, with this format:
+
+   ```
+   arn:aws:elasticloadbalancing:us-east-2:123456789012:targetgroup/my-targets/1234567890123456
+   ```
+
+1. Use the [create\-listener](https://docs.aws.amazon.com/cli/latest/reference/elbv2/create-listener.html) command to create a listener for your load balancer with a default rule that forwards requests to your target group. The listener port and protocol must match the Kubernetes service. However, TLS termination is permitted. [[double check it works in this configuration?]]
+
+   ```
+   aws elbv2 create-listener --load-balancer-arn loadbalancer-arn --protocol TCP --port 80  \
+   --default-actions Type=forward,TargetGroupArn=targetgroup-arn
+   ```
+
+## Create TargetGroupBinding CRD
+
+1. Create the [TargetGroupBinding CRD](/guide/targetgroupbinding/targetgroupbinding.md)
+
+Insert the ARN of the Target Group, as created above.
+
+Insert the name and port of the target Kubernetes service.
+
+```yaml
+apiVersion: elbv2.k8s.aws/v1beta1
+kind: TargetGroupBinding
+metadata:
+  name: my-tgb
+spec:
+  serviceRef:
+    name: awesome-service # route traffic to the awesome-service
+    port: 80
+  targetGroupARN: <arn-to-targetGroup>
+```
+2. Apply the CRD
+
+Apply the TargetGroupBinding CRD CRD file to your Cluster.
+
+`kubectl apply -f my-tgb.yaml`
+
+## Verify
+
+Wait approximately 30 seconds for the LBC to update the load balancer.
+
+[View all target groups](https://console.aws.amazon.com/ec2/v2/home#TargetGroups:) in the AWS console.
+
+Find the target group by the ARN noted above, and verify the EKS Cluster IP addresses have beened added.
```

Comment:
By cluster IP addresses, did you imply the ip addresses for the pods in the cluster? The cluster IP is a bit confusing since it could also refer to the k8s service cluster IP.

Our instructions create target group of type instance, we should be consistent here with the command. Either create target group of IP target type, or modify this line :)